### DISCUSSION TITLE
Aqara P1 Motion sensor stays Occupied

### DISCUSSION THREAD

**Discussion by gegy1** [Koenkk/zigbee2mqtt#13210] [Devices]
I paired a new Aqara P1 to Zigbee2Mqtt, but it seems it is a fault sensor or am i doing something wrong because the motion sensor stays in „Occupied“ state in Zigbee2Mqtt, even if there is no person in the room. Does anyone else have this issue?
Many thanks
Georg

**Comment by redpixy** [comment]
Were there any fans turned on in the room? Try turning the fans off and test the sensor again.

**Reply by gegy1** [reply]
Hey. No, the Room is a garage without any fans. I also tested it now in other rooms. Same result.

**Reply by redpixy** [reply]
you could try setting the detection_interval to 3 secs and motion_sensitivity to low then try testing it out again. if its continue to get stuck you may have a defective unit.

**Comment by sk8er000** [comment]
Same here I paired today 2 sensor and only one of them work properly, the other is stuck on occupied, if I set the interval to 1 second sometimes becomes unoccupied for a fraction of a second.
I also tried to place the 2 sensors on the same place but no changes on the behaviour

**Reply by gegy1** [reply]
Meanwhile i returned the sensor and bought another one from IKEA. This sensor works for me. I think the P1 is not ready for the public.

**Comment by JohnMcLear** [comment]
TS0601  _TZE200_7hfcudw5 did this for me, I had to click reconfigure and it cleared it.
